librz/cons/cons.c

```diff
@@ -1271,6 +1271,18 @@ RZ_API void rz_cons_strcat(const char *str) {
 	}
 }
 
+/**
+ * \brief Prints \p str to the RzCons, and ownership is transferred into.
+ */
+RZ_API bool rz_cons_print_own(RZ_NULLABLE char *str) {
```

Comment:
When ownership is transferred, should we add `RZ_OWN` to the argument as well? `RZ_OWN RZ_NULLABLE char *str`? cc @thestr4ng3r

Reply:
yes, I think so

Reply:
However... I wonder why we need this API. We don't usually have these kind of APIs and it looks it is just used to make the thing easier to write.

I would rather just use rz_cons_print and have the free where the string is defined.